lib/librato/rails/configuration.rb

```diff
@@ -46,14 +49,30 @@ def configure_with_config_file
         env_specific = YAML.load(ERB.new(File.read(config_file)).result)[::Rails.env]
         if env_specific
           settable = CONFIG_SETTABLE & env_specific.keys
-          settable.each { |key| self.send("#{key}=", env_specific[key]) }
+          settable.each do |key|
+            value = env_specific[key]
+            value.symbolize_keys! if key == "tags"
+            self.send("#{key}=", value)
+          end
         end
       end
 
       def default_suites
         super + DEFAULT_SUITES
       end
 
+      def default_tags
+        {
+          service: ::Rails.application.class.to_s.split("::").first.underscore,
```

Comment:
Yes, otherwise [this test](https://github.com/librato/librato-rails/blob/feature/md/test/unit/configuration_test.rb#L49) should fail if we're unable to detect the `service` name. Rails 5 is not actually in our build matrix yet, but that test passes when manually running it against `5.x` as well.

This originally could have been simplified:

```ruby
::Rails.application.class.parent_name.underscore
```

However, Rails 3.1 is the only version that did not support that approach.